model_compression_toolkit/core/common/quantization/node_quantization_config.py

```diff
@@ -460,13 +460,13 @@ def get_attr_config(self, attr_name: Union[str, int]) -> WeightsAttrQuantization
             if len(attrs_with_name) == 1:
                 attr_cfg = [v for v in attrs_with_name.values()][0]
             elif len(attrs_with_name) > 1:
-                Logger.warning(f"Found multiple weight attributes containing the name {attr_name}: "
+                Logger.warning(f"Found multiple weight attributes containing the name {attr_name}: "  # pragma: no cover
```

Comment:
I'm not sure that this shouldn't be tested because this is a warning, so there is some expected behavior that we excpect

Reply:
OK. I'll do it in the next PR.